docs/dialect.md

```diff
@@ -173,3 +173,99 @@ Allowable date parts: year, quarter, month, dayofyear, day, weekday, week, hou
 
   **Join Capabilities**  <br/>
   Join usage is also defined by capabilities in the manifest file. See the Query section for join relation capabilities [here]({{ site.baseurl }}/docs/capabilities).
+
+### Boolean Support:
+  Some databases need to customize boolean support functions.  A common case is when a database lacks native boolean support. <br/>
+   **format-true** <br/> 
+  String literal or expression for boolean false value.  
```

Comment:
Only predicate customization supported pre-2020.3.  Remove mention of literal in both format-true and format-false

Reply:
Removed the mention of literal support for  pre-2020.3

Reply:
![boolean_support](https://user-images.githubusercontent.com/14967333/85333338-44273f80-b48e-11ea-9303-413e7ad6e945.PNG)
![temp_tables](https://user-images.githubusercontent.com/14967333/85333340-44bfd600-b48e-11ea-87d6-de46ba847104.PNG)